dockerfiles/reverse-proxy/templates/nginx.conf.erb

```diff
@@ -12,10 +14,12 @@ http {
   include       /etc/nginx/mime.types;
   default_type  application/octet-stream;
 
-  log_format main '$proxy_protocol_addr - [$time_local] '
-                  '"$request" $status $body_bytes_sent '
-                  '"$http_referer" "$http_user_agent"';
-  access_log    /var/log/nginx/access.log main;
+  js_include maskQueryParams.js;
+  js_set $request_uri_masked maskQueryParams;
+
+  log_format main <%= log_format %>;
```

Comment:
I'm not super happy with this way of setting the log format, because it messes up the formatting in the `nginx.conf` file:
![image](https://user-images.githubusercontent.com/12392541/87903013-6668a900-ca9e-11ea-970d-277a01594dae.png)

I'm not sure how much of an issue that will be, since it's the generated file we probably won't be looking at too often, but if someones knows of a way to insert a multi line Ruby string via ERB and maintain formatting, let me know 😄 .